packages/aws-fargate/register.mjs

```diff
@@ -0,0 +1,19 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+/**
+ * ESM hooks supplied via loaders (--experimental-loader=@instana/collector/esm-loader.mjs) run in a dedicated thread from v20 and above,
+ * isolated from the main thread. Reference: https://github.com/nodejs/node/pull/44710
+ * We loaded the tracer in the main thread with --import, we can extend the ESM hook with register method
+ * in the future to extend the ESM support.
```

Comment:
This whole description is old and needs to be changed.
And it does not describe how register & import works.

Reply:
Please also update the esm-loader files.